gtests/test_nopython_callprofiler.cpp

```diff
@@ -212,5 +212,30 @@ TEST_F(CallProfilerTest, cancel)
     EXPECT_EQ(radix_tree().get_unique_node(), 0);
 }
 
+TEST_F(CallProfilerTest, test_serialization)
+{
+    pProfiler->reset();
+
+    foo1();
+
+    // Example:
```

Comment:
Could you please use a fixture (which can be just a string in this test file) to compare with the serialization results?

Reply:
Does this example comment block have the same content as your fixture string?  If yes, this comment block should be removed.  And as mentioned above, the fixture strings should use good indentation.